testsuite/tests/properties/call_params/pkg.adb

```diff
@@ -0,0 +1,71 @@
+package body Pkg is
```

Comment:
Is there a difference between `pkg.adb` and `callp.adb` in terms of what you're testing? (Just our of curiosity, in case I missed something)